apf/CMakeLists.txt

```diff
@@ -73,6 +74,7 @@ set(HEADERS
   apfField.h
   apfFieldData.h
   apfNumberingClass.h
+  apfComplex.h
```

Comment:
This and any other new public header will need to be added to the trililinos cmake source file list:
https://github.com/SCOREC/core/blob/master/apf/pkg_tribits.cmake

Reply:
Just resolved this in 99423ccce83a9e925dba82bc72992e96a549d4f5